pyqtgraph/functions.py

```diff
@@ -222,8 +223,8 @@ def mkColor(*args):
      float           greyscale, 0.0-1.0
      int             see :func:`intColor() <pyqtgraph.intColor>`
      (int, hues)     see :func:`intColor() <pyqtgraph.intColor>`
-     "RGB"           hexadecimal strings; may begin with '#'
-     "RGBA"          
+     "RGB"           hexadecimal strings; deprecated if they don't start
+     "RGBA"          with "#"
```

Comment:
Can we change the documentation to
```
"#RGB"     hexadecimal strings prefixed with '#'
"#RGBA"    previously allowed use without prefix is deprecated and will be removed in 0.13
```
The current version seems like too much of a double negative.

Reply:
Good point, docs shouldn't show how to not do it haha.